scipy/signal/signaltools.py

```diff
@@ -361,34 +366,59 @@ def fftconvolve(in1, in2, mode="full"):
     in1 = asarray(in1)
     in2 = asarray(in2)
 
+    if axes is None:
+        pass  # skip next few checks
+    elif not axes:
+        raise ValueError('if axes is specified it must be non-empty')
+    elif min(axes) < 0:
+        raise ValueError("axes cannot be negative")
```

Comment:
... also, negative axes should be acceptable up to the limit `-x.ndim`. Presumably these are supported by `(i)ffn` but even if not, we could wrap to their positive counterparts (assuming none are "too negative" to be correct)